libmachine/provision/debian.go

```diff
@@ -44,7 +44,7 @@ func (provisioner *DebianProvisioner) Service(name string, action pkgaction.Serv
 		return err
 	}
 
-	command := fmt.Sprintf("sudo systemctl %s %s", action.String(), name)
+	command := fmt.Sprintf("sudo systemctl %s %s || true", action.String(), name)
```

Comment:
I added a simple fix for the locale warnings, so here's the updated output (much easier to see what's going on):
```console
docker@docker:~$ sudo systemctl enable docker
Synchronizing state for docker.service with sysvinit using update-rc.d...
Executing /usr/sbin/update-rc.d docker defaults
Executing /usr/sbin/update-rc.d docker enable
docker@docker:~$ echo $?
0
```